CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/resources/data-access-view.css

```diff
@@ -5,11 +5,13 @@
     -fx-cell-hover-color: -fx-shadow-highlight-color;
 }
 
+
```

Comment:
The only changes in this file are whitespace changes., so I would suggest undoing the changes. Should not include this file in the check-in.